src/20240312_Multi_StablecoinHarmonization/StablecoinHarmonization.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "stablecoin harmonization"
+author: "Aave Chan Initiative"
+discussions: "https://governance.aave.com/t/arfc-stablecoin-harmonization-and-asset-parameters-optimization/16802"
+snapshot: "https://snapshot.org/#/aave.eth/proposal/0x22407c9362bc3294e3ddd5428fdd5c08312459595573a864ec8ebac61ad95b94"
+---
+
+## Simple Summary
+
+This AIP proposes focusing stablecoin collateral usage on USDT and USDC while setting LTV to zero for less-trafficked stablecoins and long-tail assets. The objective is to minimize risk for the protocol, optimize revenue generation, and align with market demand trends.
+
+## Motivation
+
+Aave’s revenue primarily comes from wETH borrowing and stablecoins borrowing. Years of User behavior data in DeFi indicates a low demand for fluctuating assets as collateral except in use-case-specific scenarios (LST/LRT loops for Weth, RPL borrow to deploy mini pools). Based on @ChaosLabs [data ](https://community.chaoslabs.xyz/aave/risk/markets/Ethereum/listed-assets/), only 2.4%, 3.7%, and 5.8% of supplied DAI, USDT, and USDC on Ethereum are being utilized as collateral. We can also note that market demand for alternative stablecoins is less sensitive to borrow rates compared to blue-chip stablecoins like USDT and USDC.
+
+The DAO governance has been lenient in onboarding some collaterals, such as KNC, which currently attracts nearly zero traction (currently used to generate 0.79$ of borrow volume) and generates minimal revenue. Additionally, the asymmetric risk presented by stablecoins, such as during the CRV event (collateral was USDC with very high LT and low LB), highlights the need for optimization considering Risk/reward.
+
+## Specification
+
+The ACI proposes the following strategy shift:
+
+1. Focus on USDT and USDC as primary stablecoin collaterals.
+2. Set LTV to zero for alternative stablecoins and long tail assets.
+3. Enable a 3%/week LT reduction for these stablecoins assets, providing ample time for users to react and keep them as passive yield sources or borrowable assets that represent 99% of current user demand and most of protocol revenue.
+4. Propose the following adjusted standardized ReserveFactor categories for Aave assets:
+
+| Collateral Type                            | Reserve Factor % |
+| ------------------------------------------ | ---------------- |
+| wETH                                       | 15%              |
+| USDC/USDT                                  | 10%              |
+| Alternative stablecoins & long tail assets | 20%              |
```

Comment:
I would mention with the exception of MAI, where RF is `95%`

Reply:
updated